src/xdocs/config_blocks.xml

```diff
@@ -150,6 +150,34 @@ switch (a)
     <section name="EmptyBlock">
       <subsection name="Description">
         <p> Checks for empty blocks. This check does not validate sequential blocks. </p>
+
+        <p> Sequential blocks won't be checked. Also, no violations for fallthrough: </p>
+        <source>
+            switch (a) {
+            case 1:                          // no violation
+            case 2:                          // no violation
+            case 3: someMethod(); { }        // no violation
+            default: break;
+            }
+        </source>
+
+        <p> This check processes LITERAL_CASE and LITERAL_DEFAULT separately: </p>
```

Comment:
This check processes LITERAL_CASE and LITERAL_DEFAULT separately, so if tokens=LITERAL_DEFAULT, following code will not trigger violation, as empty block belongs LITERAL_CASE: